### ISSUE TITLE
Add tip about /etc/exports

### ISSUE THREAD

**Issue by ericclemmons** [nlf/dlite#29]
As I had vagrant machines, `dlite daemon` failed because `/etc/exports` already had entries, preventing mounting.

**Comment by nlf** [comment]
interesting. did it already have an entry to share /Users specifically? because if not i don't think it should've stopped dlite from starting. do you recall what the error message was?

**Comment by ericclemmons** [comment]
Since it was from Vagrant, it was `/Users/Eric/Projects/org/project` being mounted.

Initially, `dlite install` "worked" fine, and `launchctl load` didn't throw any errors.  I didn't see any `/var/run/docker.lock` file, so I started digging by running `dlite daemon` and _that's_ when I saw a note about `/Users` and exports.

I wish I saved the error, but it was, like, 3AM and Docker plays tricks on you.
